custom/enikshay/integrations/ninetyninedots/api_spec.py

```diff
@@ -59,17 +65,49 @@ class DotsApiParamChoices(DotsApiSectorParam):
 
 
 class DotsApiParam(StrictJsonObject):
+    """99DOTS <-> eNikshay API Parameter Definition
+
+    This class defines api parameters for the patient details API between
+    99DOTS and eNikshay.
+
+    For incoming api requests from 99DOTS, it defines where and how to save
+    parameters.
+
+    For outgoing api requests to 99DOTS, it defines which properties to watch
+    for changes to and how they are compiled.
+
+    """
+
+    # the parameter name for the json sent and received
     api_param_name = jsonobject.StringProperty(required=True)
+
+    # whether this parameter is required when receiving and API request
     required_ = jsonobject.BooleanProperty(default=False, name='required')
```

Comment:
nit: why using underscore here?

Reply:
I think `required` is already taken for jsonobjects